### DISCUSSION TITLE
Transcription

### DISCUSSION THREAD

**Discussion by MonicaGalassi** [MukurtuCMS/Mukurtu-CMS#27] [Feature Enhancement]
@michael-wynne-wsu I find that the transcription section in the DHI could be more visible (in our latest sites users need to click on the "transcription" section to visualise the long text). Also, I think that having the option to integrate a transcription tool would be of great help for communities wanted to transcribe the records themselves or update the current narratives.

**Comment by michael-wynne-wsu** [comment]
We are looking into integration of OHMS (Oral History Metadata Synchronizer https://www.oralhistoryonline.org/) as a more powerful transcription tool. But certainly open to exploring other options as well.

If you want to change the current transcription field to NOT be collapsed by default... (see attached screenshots)
- go to /admin/structure/types/manage/digital-heritage/display/full
- change the transcription fieldgroup settings from "collapsed" to "open" or "collapsible"
- Update > Save
Contact support by email if you want more help with this though
![Screen Shot 2022-06-16 at 11 20 26 AM](https://user-images.githubusercontent.com/88502274/174139437-b6396efd-69c4-420a-beed-0f1497a69ffb.png)
![Screen Shot 2022-06-16 at 11 20 33 AM](https://user-images.githubusercontent.com/88502274/174139444-d9cbb9a1-f0e8-48c3-aeec-e5bed6db8c49.png)
![Screen Shot 2022-06-16 at 11 20 49 AM](https://user-images.githubusercontent.com/88502274/174139445-c6345c9a-ee8c-4292-9a23-36aab5f16343.png)
!
